docs/api/en/materials/MeshStandardMaterial.html

```diff
@@ -49,7 +49,7 @@ <h1>[name]</h1>
 		</p>
 		<p>
 			Technical details of the approach used in three.js (and most other PBR systems) can be found is this
-			[link:https://disney-animation.s3.amazonaws.com/library/s2012_pbs_disney_brdf_notes_v2.pdf paper from Disney] (pdf),
+			[link:https://media.disneyanimation.com/uploads/production/publication_asset/48/asset/s2012_pbs_disney_brdf_notes_v3.pdf paper from Disney] (pdf),
```

Comment:
Do you mind updating the `zh` doc page, too?

https://github.com/mrdoob/three.js/blob/6cfe91706d1b5434f250d7be76cad29287926f58/docs/api/zh/materials/MeshStandardMaterial.html#L40

Reply:
@Mugen87 sure, I added commit to this PR.